clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -461,13 +463,77 @@ check_function_markers()
     return true;
 }
 
+bool
+check_repeated_syscall_with_same_pc()
+{
+    constexpr addr_t ADDR_ONE = 0x7fcf3b9dd9e9;
+    // Negative: syscalls with the same PC.
+#if defined(X86_64) || defined(X86_32) || defined(ARM_64)
+    {
+        std::vector<memref_t> memrefs = {
+            gen_marker(1, TRACE_MARKER_TYPE_FILETYPE, OFFLINE_FILE_TYPE_ENCODINGS),
+#    if defined(X86_64) || defined(X86_32)
+            gen_instr_encoded(0, { 0x01 }),
```

Comment:
Why do we need this first instr? Why does it not return a pc discontinuity error for the unexplained jump from pc=0 to ADDR_ONE? (Not sure what the decoded instr is; pls add it to the comment like for the others)

Reply:
Removed it.

Reply:
Removed it altogether.